File: csharp/src/Behavioral/Strategy/DvdNameReplaceSpacesStrategy.cs
namespace Behavioral.Strategy {


//DvdNameReplaceSpacesStrategy.java - three of three concrete strategies

public class DvdNameReplaceSpacesStrategy : DvdNameStrategy {
   override public string formatDvdName(string dvdName, char charIn) {
       return dvdName.Replace(' ', charIn);
   }
}

}

File: scala/src/main/scala/Behavioral/Strategy/DvdNameReplaceSpacesStrategy.scala
package Behavioral.Strategy;


//DvdNameReplaceSpacesStrategy - three of three concrete strategies

public class DvdNameReplaceSpacesStrategy extends DvdNameStrategy {
   public String formatDvdName(String dvdName, char charIn) {
       return dvdName.replace(' ', charIn);
   }
}